iOS/DuckDuckGo/PrivacyDashboard/PrivacyDashboardViewController.swift

```diff
@@ -318,6 +332,9 @@ extension PrivacyDashboardViewController {
         let webVitalsResult = await calculateWebVitals(breakageAdditionalInfo: breakageAdditionalInfo,
```

Comment:
I think we can delete this right? (this is a second call into the JS on breakage reports).

`jsPerformance` should be a property of the expanded right?

Reply:
Yeah it is, reason I hadn't changed this is because I was wary of introducing regressions since this is unchanged.  Will swap it out tomorrow - should be fairly safe and we skip a call.